lib/train/transports/ssh.rb

```diff
@@ -63,6 +63,7 @@ class SSH < Train.plugin(1) # rubocop:disable Metrics/ClassLength
     option :bastion_user, default: 'root'
     option :bastion_port, default: 22
     option :non_interactive, default: false
+    option :verify_host_key, default: false
```

Comment:
Funny, I just hit a deprecation issue on this option's value with net-ssh v5.  Once we pull this in I'll adjust #430 .